pom.xml

```diff
@@ -80,17 +80,6 @@
       <artifactId>joda-time</artifactId>
       <version>2.9.5</version>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci</groupId>
-      <artifactId>annotation-indexer</artifactId>
-      <version>1.12</version>
-    </dependency>
-    <dependency>
-      <groupId>com.infradna.tool</groupId>
-      <artifactId>bridge-method-annotation</artifactId>
```

Comment:
Also noticed these two dumped in the archive, which is unnecessary as they are `provided` by the parent POM.